src/autocomplete.js

```diff
@@ -88,6 +100,9 @@ class Autocomplete {
     }
 
     $init() {
+        /**
```

Comment:
why do you use multiline syntax in these cases?

Reply:
It's just IDE's auto-insertion of snippet. In case of one-liners I manually changed those.

Reply:
Done